operator/apis/platform/v1alpha1/central_types.go

```diff
@@ -187,6 +199,15 @@ type DeclarativeConfiguration struct {
 	Secrets []LocalSecretReference `json:"secrets,omitempty"`
 }
 
+// NotifierSecretsEncryption defines settings for encrypting notifier secrets in the Central DB.
+type NotifierSecretsEncryption struct {
+	// Enables the encryption of notifier secrets stored in the Central DB. An encryption key must be
+	// provided in a secret called `central-encryption-key` in the Central namespace.
```

Comment:
Not sure if the entire filepath will be configurable or has to be correct at the secret level. But for now it looks like they assume a specific filepath, so we could also include that the data field in the secret containing the key should be called `encryptionKey`

Reply:
Done